app/Http/Controllers/MissingController.php

```diff
@@ -64,16 +64,16 @@ public function index_gamescreens(){
     }
 
     //Spiele mit fehlenden Spieledateien anzeigen
-    public function index_gamefiles(){
+    public function index_gamefiles() {
```

Comment:
__[code_sniffer]__
* Method name "MissingController::index_gamefiles" is not in camel caps format
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->